tests/unit/CmsZone.spec.ts

```diff
@@ -124,8 +123,8 @@ describe('CmsZone.vue', (): void => {
       await response;
       await localVue.nextTick();
       await localVue.nextTick();
-      const expectedClasses = zoneType === 'scrolling' ? ['scrollable-content'] : [];
-      expect(wrapper.classes()).toEqual(expectedClasses);
+      await localVue.nextTick();
```

Comment:
Might be useful to copy in this `settled` helper that we use in freshebt-vue and freshcard-app:

```ts
// NOTE(mohan): This helper makes sure all promises are resolved and everything is rendered.
// This is useful in cases where you have to call Vue.nextTick many times.
// Unceremoniously copied from: https://github.com/kentor/flush-promises/blob/master/index.js.
export async function settled(): Promise<void> {
  return new Promise(function (resolve) {
    setTimeout(resolve);
  });
}
```

and then you can replace all the nextTicks with one `await settled();`

Reply:
I tried that and couldn't get all the tests to pass :( . happy to pair if you can help me power through!

Reply:
Oh weird... maybe `settled` isn't as universal as i thought. Let's pair on it later!